docs/installation.md

```diff
@@ -27,74 +27,85 @@ Here is a command to create and activate a new virtual environment with *conda*
 ```py
 conda create -p <path_to_enviroment> python=3
 conda activate <path_to_environment>
```

Comment:
```suggestion
conda create --name <my-env>
conda activate <my-env>
```
I suggest replacing this with this suggestion to be more consistent with the [conda docs](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) and also with my experience of what people typically do

Reply:
This is an interesting point. It is indeed more common in documentation to use named environments. However, I think it's (much) better practice to use path-based environments, for a variety of reasons mostly related to reliability and reproducibility. But perhaps in the case of these docs, the easiest thing to do is just fit in with what's commonplace. Another option would be to include both options somewhere.

Some reasons to prefer path-based environments are as follows:
 - Named environments go into the users home directory, which frequently fills up and has issue with disk space quotas (this is by far the most common issue)
 - On shared systems, overall usage across multiple users can fill up due to large numbers of virtual environments being created in home directories, even if no one user hits their quota
 - Home directories are more likely to be impacted by OS and version updates, whereas a path-based environment is more isolated from these changes
 - Path-based environments can be more easily activated or used by other software, or executed using a fully-qualified path to the Python environment in scripts. The use of name-based environments require conda to be activated in the bashrc / current execution context, which is not always true.
 - Some users have more than one installation of conda itself, and using a path-based approach allows people to be more flexible in using multiple versions of conda and multiple environments. 
 - If building an environment on behalf of someone else, it's easier to give them a path-based environment to use
 - If a sysadmin needs to help clean up a full partition, it's easier to give them a path/directory to delete

It's true these are probably more advanced (or just more sysadmin-focused) use cases, but they are commonly encountered in shared computing environments. Name-based environments are kind of the "default" and typically seen in documentation, however they don't seem (to me) to have any specific advantages.

Reply:
Yes I agree that there are a lot of advantages to specifying the path. I've taken both approaches depending on the machine I'm working on.

If someone is aware of these issues then they will probably be a more advanced user who is aware of different ways to install conda. If not, then there is a chance that they won't install it in a sensible location or won't know where to install it. 

The suggestion to change this was just to keep the install instruction as simple as possible and consistent with most other docs.

There are other changes to this process that I would do in practice, but may be a distraction, such as always using mamba instead of conda.

Happy either way as to if this changes or stays the same.

Reply:
Thanks @nicholasloveday - I have included your changes